pdns/rpzloader.cc

```diff
@@ -317,11 +323,18 @@ void RPZIXFRTracker(const ComboAddress& master, boost::optional<DNSFilterEngine:
     }
   }
 
+  bool skipRefreshDelay = isPreloaded;
```

Comment:
can't we deduce `isPreloaded` from `sr != nullptr` earlier in `RPZIXFRTracker`? That saves us a parameter.